kima/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "kima",
+  "assets": [
+    {
+      "description": "The native token of Kima Chain",
+      "denom_units": [
+        {
+          "denom": "uKIMA",
+          "exponent": 0
+        },
+        {
+          "denom": "KIMA",
+          "exponent": 6
+        }
+      ],
+      "base": "uKIMA",
+      "name": "KIMA",
+      "display": "KIMA",
+      "symbol": "KIMA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/kimatestnet/images/kima.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/kimatestnet/images/kima.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/kimatestnet/images/kima.png",
```

Comment:
Please point this to the mainnet's images directory, and update the testnet assets to be test-mintages of this asset (and remove the kima.png,svg images from the testnet directory).

Reply:
okay